compose.langflow.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  langflow:
+    container_name: ${HARBOR_CONTAINER_PREFIX:-harbor}.langflow
+    image: langflowai/langflow:${HARBOR_LANGFLOW_VERSION:-latest}
+    ports:
+      - "${HARBOR_LANGFLOW_HOST_PORT:-7860}:7860"
```

Comment:
The default values/env are all defined in the profiles/default.env, it gets merged into .env during install, and updates (use `harbor config update` to merge in-dev)